src/vos/README.md

```diff
@@ -666,63 +666,8 @@ For reference, key junction points in the flows are:
 
 ## Metadata Overhead
 
-VOS provides a tool vos_size.py that can take a set of assumptions about how many keys and objects and VOS pools are in use and provide an estimate of metadata overhead.
+There is available a tool to estimate the metadata overhead. It is described on the <a href="https://github.com/daos-stack/daos/blob/master/src/client/scm_estimator/README.md">scm_estimator</a> section.
```

Comment:
If you should have to push another commit for any other reason:

```suggestion
There is a tool available to estimate the metadata overhead. It is described on the <a href="https://github.com/daos-stack/daos/blob/master/src/client/scm_estimator/README.md">scm_estimator</a> section.
```

Reply:
Done

Reply:
@berserk-fury You should leave conversations to be resolved by the person that started them.  That way the person who made the comment/request closes the loop on whether their concern was addressed or not.

Reply:
That's make sense. I'm curious about the workflow. After fixing it, should I reply to the individual conversation loops. Or is it enough to only re-request the code review of the whole PR?

Reply:
> That's make sense. I'm curious about the workflow. After fixing it, should I reply to the individual conversation loops. Or is it enough to only re-request the code review of the whole PR?

It's up to you.  It's useful to see that the author has ack'd the comment/request and acted on it (or not, because they disagree perhaps), but really, a review re-request should be sufficient.  The reviewer can then go back and look at their open conversations and see if their issues were resolved in the new patches on the PR.

FWIW, the policy on force-push is in https://wiki.hpdd.intel.com/display/CI/The+New+CI+Pipeline.  It's probably worth reviewing that page if you have not already.  If the bit about resolving conversations is not there, I will add it.